package.json

```diff
@@ -28,49 +28,44 @@
     "lint:src": "eslint \"src/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
     "lint:tests": "eslint \"**/__tests__/**/*.ts\"",
     "test": "npm run test:unit && npm run test:system",
-    "test:system": "env-cmd __tests__/__resources__/env/system.env jest .*/__system__/.* --coverage false --runInBand",
-    "test:unit": "env-cmd __tests__/__resources__/env/unit.env jest --coverage --testPathIgnorePatterns \".*/__system__/.*\" ",
+    "test:system": "env-cmd -f __tests__/__resources__/env/system.env jest .*/__system__/.* --coverage false --runInBand",
+    "test:unit": "env-cmd -f __tests__/__resources__/env/unit.env jest --coverage --testPathIgnorePatterns \".*/__system__/.*\" ",
     "installPlugin": "npm install && npm run clean && npm run build && bright plugins install .",
     "typedoc": "typedoc --out ./docs/typedoc/ ./src/ --disableOutputCheck"
   },
   "imperative": {
     "configurationModule": "lib/imperative.js"
   },
   "dependencies": {
-    "ibm_db": "3.2.2"
+    "ibm_db": "3.2.3"
   },
   "peerDependencies": {
-    "@zowe/imperative": "^5.0.0"
+    "@zowe/imperative": ">8.0.0-next.0 <9.0.0"
```

Comment:
I think we have gone back and forth on this, but curious if the intention is to allow for any `8.x.x` (non-prerelease included) to be compatible with this version. 🤔

Reply:
Not really, I plan for this to update when we are out of prerelease to be >8.0.0.